jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/command/CaseCommand.java

```diff
@@ -71,4 +75,14 @@ protected void triggerRules(KieSession ksession) {
         }
         ksession.fireAllRules();
     }
+    
+    protected CaseFileInstance getCaseFile(KieSession ksession, String caseId) {
+        Collection<? extends Object> caseFiles = ksession.getObjects(new ClassObjectFilter(CaseFileInstance.class));
+        if (caseFiles.size() == 0) {
+            return null;
+        }
+        CaseFileInstance caseFile = (CaseFileInstance) caseFiles.iterator().next(); 
```

Comment:
Can be more than a single case file returned by `ksession.getObjects`?

Reply:
yes, when wrong strategy is used

Reply:
In that case - just as a defensive approach, I would rather throw appropriate exception for that.

Reply:
we don't do defensive programming ... plus to do this it would have to have yet another if that has very limited value in most of the cases